playbooks/roles/suse-build-images/defaults/main.yml

```diff
@@ -65,7 +67,7 @@ docker_images: []
 # suse_osh_deploy_libvirt_yaml_overrides:
 #   images:
 #     tags:
-#       libvirt: "<insert build server url here, probably local-registry:5000>/openstackhelm/libvirt:latest-opensuse_15"
+#       libvirt: "{{ build_registry }}openstackhelm/libvirt:latest-opensuse_15"
```

Comment:
Actually `{{ build_registry }}` is not known at this step, so until we fix it, there needs to be full url, like `$USERNAME-osh.openstack.local:5000`